docs/schema.json

```
@@ -37,15 +37,11 @@
             "handlers": {
               "title": "The handlers global configuration.",
               "markdownDescription": "https://mkdocstrings.github.io/handlers/overview/",
-              "type": "object",
-              "default": null,
-              "items": {
-                "oneOf": [
-                  {
-                    "$ref": "https://raw.githubusercontent.com/mkdocstrings/python/master/docs/schema.json"
-                  }
-                ]
-              }
+              "anyOf": [
+                {
+                  "$ref": "https://raw.githubusercontent.com/mkdocstrings/python/master/docs/schema.json"
```

Oh let me take this opportunity to switch from `master` to `main` too:

```suggestion
                  "$ref": "https://raw.githubusercontent.com/mkdocstrings/python/main/docs/schema.json"
```